VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participant-display-name/private-consultation-participant-display-name.component.scss

```
@@ -0,0 +1,7 @@
+.white {
+  color: rgba(255, 255, 255, 0.9);
+}
+
+.margin-right {
+  margin-right: 5px;
```

Could you change this to use rem. You can use the following sass function but you will need to import the abstracts first
```suggestion
  margin-right: rem(5px);
```

---

@FarronF Sure, I've updated it as suggested